database/migrations/2016_06_26_225535_create_favourites_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class CreateFavouritesTable extends Migration {
+
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		Schema::create('favourites', function(Blueprint $table)
+		{
+			$table->increments('id');
+			$table->integer('user_id')->unsigned()->index();
+			$table->integer('track_id')->unsigned()->nullable()->index();
+			$table->integer('album_id')->unsigned()->nullable()->index();
+			$table->integer('playlist_id')->unsigned()->nullable()->index();
+			$table->timestamp('created_at')->default(DB::raw('CURRENT_TIMESTAMP'))->nullable();
```

Comment:
I'm seeing `timestamp` and `dateTime` being set as the column type in different migrations, and `CURRENT_TIMESTAMP` and `now()` both being used in places as well. Are they interchangeable?